src/app/data/components/data-browser/helper-classes/filter-parser.ts

```diff
@@ -294,6 +297,17 @@ export class DataBrowserFilterParser {
             appendTo.push(JSON.stringify(blacklist));
     }
 
+    private appendBlackAndWhiteListComments(appendTo: string[], searchElement: any): any {
+        let element = {
```

Comment:
might be mistaken but don'T we need a check for active here somewhere?

Reply:
Yes, I added the condition for active